vk-page-header/package/class-vk-page-header.php

```
@@ -752,6 +757,105 @@ public function save_custom_fields() {
+			// テーマかプラグインかに応じて URL のベースを決定する。
+			$script_url = '';
+			$theme_dir  = get_template_directory();
+			if ( strpos( $script_dir, $theme_dir ) !== false ) {
+				// Inside a theme.
+				// テーマ内の場合。
+				$relative   = str_replace( $theme_dir, '', $script_dir );
+				$script_url = get_template_directory_uri() . $relative . '/js/vk-page-header-panel.min.js';
+			} else {
+				// Inside a plugin or wp-content direct.
+				// プラグインまたは wp-content 直下の場合。
+				$content_dir = WP_CONTENT_DIR;
```

PHPStan (2.1.46)

[error] Constant WP_CONTENT_DIR not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)